### ISSUE TITLE
Weird Cert on Windows Installer

### ISSUE THREAD

**Issue by NickETH** [jupyterlab/jupyterlab-desktop#648]
## Description

The cert, which is used to sign the window installer can not be verified!
Please do not sign this installer like this.
Either use a cert from a well known vendor, which roots are widely distributed
Or do not sign at all!
A signature like this produces a lot more questions as it is able to answer.

## Reproduce
Just download the Windows Installer.
Powershell:
Get-AuthenticodeSignature -FilePath "JupyterLab-Setup-Windows.exe"

</details>

**Comment by JasonWeill** [comment]
What certificate do you see, and which certification vendors would you prefer be used instead?

**Comment by JasonWeill** [comment]
For example, are there other open source projects that you use on your Windows installation that use signed installers? If so, which certification authorities do they use?

**Comment by NickETH** [comment]
You are using a developer cert from Apple. This seems to be self signed.
There is no chain of trust behind this like on a real cert.
![image](https://user-images.githubusercontent.com/35805296/234359653-62fdaba6-f8c7-4f31-90b4-de49240fb9da.png)
You can use a code signing cert from any vendor. 
We use one from Digicert e.g.
Thanks

**Comment by krassowski** [comment]
I think this is a duplicate of https://github.com/jupyterlab/jupyterlab-desktop/issues/202 which tracks code signing on Windows (and Mac - this one was done).

@mbektas what is blocking us here? Is there something we could do to unblock this, i.e. is this something we need to take up to the Jupyter EC to get a certificate paid for, or is this a software problem to be solved?

**Comment by NickETH** [comment]
You will definitely need Code Signing Cert for Windows.
Should be not that expensive for open source (30-80$/y).

**Comment by mbektas** [comment]
@krassowski NumFocus was going to purchase a Windows signing certificate and pass the information to me last time we discussed. We haven't synced on that for a very long time.

**Comment by NickETH** [comment]
Any news on this one?

**Comment by mbektas** [comment]
I am working with NumFOCUS to get a signing certificate purchased.
Is this a blocking issue for you? Our Apple signing certificate is not self-signed and we have been using it to sign macOS application. It turns out our build tool (electron builder) is using it also to sign the Windows binaries.

**Comment by JasonWeill** [comment]
Resolving as a duplicate of #202.
